src/test/testdata/lsl/literalTypes.lsl

```diff
@@ -0,0 +1,23 @@
+libsl "1.0.0";
+library literalTypes;
+
+typealias Int = int32;
+typealias Five = 5;
+
+automaton A : Int {
+    fun *.localVariablesWithLiteralTypes(): 6 {
+        var a: 5;
+        val stringType_1: "hello";
```

Comment:
As I understand it, this is a variable that have not been initialized but have a type `string` with a single allowed value `"hello"`. What it contents would be? What its "default" value would be?